### DISCUSSION TITLE
Footer query

### DISCUSSION THREAD

**Discussion by karani12** [vuepress-theme-hope/vuepress-theme-hope#1616] [Q&A] [answered]
Hi, I would like to know if there is a way I could do a corprate footer like say [this](https://www.freecodecamp.org/).

I want to do a footer that has more information. Is there a slot for footer where I can span the whole width.

Another footer example would be https://supabase.com/docs

In default theme we have slots to do this but i'm not sure if there is a way to do this here.

Thanks

**Comment by Mister-Hope** [comment, marked as answer]
https://vuepress-theme-hope.github.io/v2/guide/advanced/customize.html#replace-components

Besides slots, you can use you pr own component.
